gprofiler/profilers/java.py

```diff
@@ -661,6 +764,16 @@ def read_output(self) -> Optional[str]:
             default=True,
             help="Disable collection of async-profiler meminfo at the end of each cycle (collected by default)",
         ),
+        ProfilerArgument(
+            "--java-jvm-flags-to-collect",
+            dest="java_jvm_flags_to_collect",
+            type=str,
+            nargs="?",
+            default=JavaFlagCollectionOptions.DEFAULT.value,
```

Comment:
Add an option to disable collection altogether. `--java-collect-jvm-flags=none`, `--java-collect-jvm-flags=default` and so on. Please rename the flag to `--java-collect-jvm-flag`, this is imperative unlike `flags-to-collect`.

Reply:
And then rename the `to_collect` name everywhere, to the new name